doc/Tribler development on Windows.md

```diff
@@ -57,7 +57,7 @@ After installation, you should set an environment variable to let libtorrent kno
 
 Next, you should build Boost.build. You can do this by opening the Visual Studio command prompt and navigating to your Boost libraries. Navigate to `tools\build` and execute `bootstrap.bat`. This will create the `b2.exe` file. In order to invoke `b2` from anywhere in your command line, you should add the Boost directory to your user PATH environment variable. After modifying your PATH, you should reopen your command prompt.
 
-Now, download the libtorrent source code from [GitHub](https://github.com/arvidn/libtorrent/releases) and extract it. Version 1.0.7 of Libtorrent seems to contain a bug when downloading torrents from the DHT so it is recommended to use the source code of version 1.0.6. Open the Developer Command Prompt shipped with Visual Studio (not the regular command prompt) and navigate to the location where you extracted the libtorrent source. In the directory where the libtorrent source code is located, navigate to `bindings\python` and build libtorrent by executing the following command (this takes a while so make sure to grab a coffee while waiting):
```

Comment:
"it is advised"
Also, it's not necessary if they are building a 32 bit version, they can just download the installer.